Participants/Sara Carvalho/hysterectomy_Sara.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.tei-c.org/release/xml/tei/custom/schema/relaxng/tei_all.rng" type="application/xml"
+	schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<?xml-model href="../../Schemas/TEILex0-ODD.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="DicEndo">
+   <teiHeader>
+      <fileDesc>
+         <titleStmt>
+            <title>Title</title>
+         </titleStmt>
+         <publicationStmt>
+            <publisher/>
+         </publicationStmt>
+         <sourceDesc>
+            <bibl/>
+         </sourceDesc>
+      </fileDesc>
+   </teiHeader>
+   <text>
+      <body>
+         <entry xml:id="DicEndo.hysterectomy" xml:lang="en">
+         <form type="lemma">
+            <orth>Hysterectomy</orth>
+            <pron><pc>/</pc>ˌhɪstəˈrektəmi<pc>/</pc></pron>
```

Comment:
You should put `<pc>` outside of `<pron>` because those slashes function as delimiters but they are themselves not part of the pronunciation

Also, you could add the attribute notation to indicate that this is ipa like this `<pron notation="IPA"></pron>`